src/lib/sql/sql_to_ast_translator.cpp

```diff
@@ -1,5 +1,6 @@
 #include "sql_to_ast_translator.hpp"
 
+#include <util/sqlhelper.h>
```

Comment:
thats not the alphabetical include order 😛

Reply:
Fixed. CLion bloody "auto-add-include"